static/styles/app_components.css

```diff
@@ -532,7 +532,7 @@ div.overlay {
     }
 
     &.stream_create_info {
-        margin: 10px 10px 0 10px;
+        margin: 10px 10px 0;
```

Comment:
Is this 3-value format actually more readable?  I at least find it hard to read, but I'm not good at reading margins.

Reply:
All you need to remember is that it goes in clock order, bouncing off both ends.

* 1 2 3 4: top=1, right=2, bottom=3, left=4
* 1 2 3: top=1, right=2, bottom=3, left=2
* 1 2: top=1, right=2, bottom=1, left=2
* 1: top=1, right=1, bottom=1, left=1

Reply:
I guess we don't actually use it in that many places in any case and can always change our mind later; I'll merge.